src/prowjobsscraper/equinix_metadata.py

```diff
@@ -47,6 +48,12 @@ def _set_equinix_metadata(self, job: ProwJob) -> None:
             logger.debug("Equinix metadata are missing from %s: %s", metadata_path, e)
 
         if raw_metadata:
-            job.equinixMetadata = EquinixMetadata.parse_raw(raw_metadata)
+            try:
+                job.equinixMetadata = EquinixMetadata.parse_raw(raw_metadata)
```

Comment:
Is there something important in this data that we need to know in case of job failure?

Reply:
This is `Equinix` related data for the job, in case we managed to parse it will gather it, otherwise skip